projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts

```diff
@@ -52,10 +49,10 @@ interface CellData {
   parser: CoreTableCellParserType.NoOp
 })
 export class ExitCallsTableCellRendererComponent extends TableCellRendererBase<CellData, Trace> implements OnInit {
-  public readonly apiCalleeNameCount: string[][];
+  public readonly apiCalleeNameEntires: [string, string][];
   public readonly apiExitCalls: number;
-  public readonly maxShowApiCalleeNameCount: number = 10;
-  public readonly totalCountOfDifferentApiCallee!: number;
+  public readonly MAX_API_CALLEE_TO_SHOW: number = 10;
```

Comment:
With a const like this, we should make it static (and lift it above the instance vars)

Reply:
also, please rename to `MAX_API_CALLEES_TO_SHOW`